Automattic/wp-calypso
# Adds name fields to the G Suite user add screen

#### Changes proposed in this Pull Request

* Adds First Name and Last Name fields to Domains -> Email -> Add User

#### Bonus changes 

* Fixes a JS error that prevented adding extra email addresses 
* Fixes a minor typo on the success page

#### Testing instructions

* Checkout branch
* https://wordpress.com/domains/manage/{DOMAIN}/add-google-apps/{DOMAIN}
* Click `Add G Suite User`
* Observe Name fields
* Fill out fields
* Continue through checkout
* Observe that name is correct in G Suite

p2MSmN-6RF-p2
